src/lib/components/IntersectionObserver/key.ts

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
Can you provide a bit of explanation somewhere about how this is getting used? I feel like I'm halfway to understanding how this is working with Svelte context, but a quick comment (or type declaration, if applicable) would help.